modules/cnv-example-storage-class-yaml.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_users_guide/cnv-configuring-local-storage-for-vms.adoc
+
+[id="cnv-example-storage-class-yaml_{context}"]
+= Example: StorageClass YAML for the hostpath provisioner
+
+The following YAML file is used to create the storage class for the hostpath
+provisioner. It includes sample values; ensure that you replace them with your
+own values where applicable. +
+[NOTE]
+====
+All values shown in this example are required.
+====
+
+[source,yaml]
+----
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: hostpath-provisioner <1>
+provisioner: kubevirt.io/hostpath-provisioner
+reclaimPolicy: Delete
+volumeBindingMode: WaitForFirstConsumer <2>
+----
+<1> You can optionally rename the storage class by changing this value.
+<2> The `volumeBindingMode` value determines when dynamic provisioning and volume
+binding occur. Specify `WaitForFirstConsumer` to delay the binding and provisioning
+of a PersistentVolume (PV) until after a Pod that uses the PersistentVolumeClaim (PVC)
+is created. This ensures that the PV meets the Pod's scheduling requirements.
+The `WaitForFirstConsumer` setting is not available in {product-title} 3.x.
```

Comment:
Should "Immediate" volumeBindingMode be mentioned and make a distinction with "WaitForFirstConsumer"?

Reply:
@qwang1 Hi Qixuan, I intentionally didn't include `Immediate` because IIUC it is only used with OpenShift 3.x, and we don't recommend using CNV 2.2 with OCP 3.x.

Reply:
@ousleyp Thanks, good to know